pyte/streams.py

```diff
@@ -129,7 +129,7 @@ class Stream(object):
 
     #: A regular expression pattern matching everything what can be
     #: considered plain text.
-    _special = set([ctrl.ESC, ctrl.CSI, ctrl.NUL, ctrl.DEL, ctrl.OSC, ctrl.ST])
```

Comment:
The problem is that symbol `➜` is three bytes `\xe2\x9e\x9c`, but _text_pattern regex assumes that `\x9c` is not a text, so streamer tries to encode just two bytes `\xe2\x9e` and fails.